diff --git a/node_modules/@mantine/nprogress/esm/NavigationProgress.js b/node_modules/@mantine/nprogress/esm/NavigationProgress.js
index bb64edc..55809f9 100644
--- a/node_modules/@mantine/nprogress/esm/NavigationProgress.js
+++ b/node_modules/@mantine/nprogress/esm/NavigationProgress.js
@@ -71,8 +71,10 @@ function NavigationProgress({
   const reset = () => {
     setUnmountProgress(true);
     stop();
-    setProgress(0);
-    window.setTimeout(() => setUnmountProgress(false), 0);
+    window.setTimeout(() => {
+      setProgress(0);
+      setUnmountProgress(false)
+    }, 0);
   };
   const complete = () => setProgress(100);
   const cancelUnmount = () => {
